src/components/VisionMissionComponent.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { Grid, createMuiTheme, responsiveFontSizes, ThemeProvider} from "@material-ui/core"
+import VisionMissionLayout from "./VisionMissionLayout"
+
+let theme = createMuiTheme();
+theme = responsiveFontSizes(theme);
+
+export default function VisionMissionComponent() {
+
+  return (
+    <ThemeProvider theme={theme}> 
+      <Grid  container justify="space-around"  style={{ height: "100vh" }}>
+        <Grid item container xs={10} sm={8}>
+          <Grid item xs={12} md={8}>
+            <VisionMissionLayout title="vision" body="Act as platform of “unity” to promote the `cause` of Computer Science. Develop a sense of belongingness amongst the pupils in the `impact` being created by Computer Science and it’s applied concepts while appreciating its inter-disciplinary nature and recognizing the unique art of developing solutions through mathematical models evolving in scale to the emerging problems of the world today. And help explore knowledge, simulate study and research of and seek opportunities in Computer Science, with special regard given to Programming, as its members pave way for their career into the future."/>
```

Comment:
Can we remove the extra Grid component?

Reply:
I could not find another alternative.  The main grid should occupy 8 columns in center and its direction should be column, one of its items should occupy 8(within the main grid) to the left and the other should occupy 8(within the main grid)  to the right.